src/NuGet.Indexing/ServiceHelpers.cs

```diff
@@ -51,29 +52,28 @@ public static void WriteResponse(IOwinContext context, ClientException e)
         }
 
 
-        public static void WriteResponse(IOwinContext context, Exception e)
+        public static void WriteResponse(IOwinContext context, Exception e, FrameworkLogger logger)
```

Comment:
`ServiceHelpers` should be removed entirely from the NuGet.Indexing project IMO :)

That's expanding on the intent of this PR though. Happy to make it happen, but would prefer small, incremental and focused changes. Intent of this PR was to focus on the proposed logging *design*, and when merged into `searchcloud`, we can optimize different aspects of the code? Extracting ServiceHelpers can be done outside of the scope of this PR.